templates/arista_eos_show_ip_helper-address.template

```diff
@@ -0,0 +1,9 @@
+Value Required INTERFACE (\S+)
+Value List IP_HELPER (\d+\.\d+\.\d+\.\d+|\S+)
+
+Start
+ ^\S -> Continue.Record
```

Comment:
Can you add `^DHCP` above L5 and `^\s+DHCP\s+Smart` above L7? And then add the following under L8?
```
 ^\s*$$
 ^. -> Error
```

Reply:
This has been corrected